.3rd-party/latest.md

```diff
@@ -1,55 +1,37 @@
+| com.amazonaws                      | aws-java-sdk-core            | 1.11.392       | [CQ17353](https://dev.eclipse.org/ipzilla/show_bug.cgi?id=17353) |
+| com.amazonaws                      | aws-java-sdk-kms             | 1.11.392       | [CQ17354](https://dev.eclipse.org/ipzilla/show_bug.cgi?id=17354) |
+| com.amazonaws                      | aws-java-sdk-s3              | 1.11.392       | [CQ17355](https://dev.eclipse.org/ipzilla/show_bug.cgi?id=17355) |
+| com.amazonaws                      | jmespath-java                | 1.11.392       | [CQ17356](https://dev.eclipse.org/ipzilla/show_bug.cgi?id=17356) |
+| com.fasterxml.jackson.dataformat   | jackson-dataformat-cbor      | 2.8.9          | []()                                                             |
+| joda-time                          | joda-time                    | 2.9.9          | []()                                                             |
+| org.bouncycastle                   | bcpkix-jdk15on               | 1.47           | []()                                                             |
+| org.bouncycastle                   | bcprov-jdk15on               | 1.47           | []()                                                             |
+| org.codehaus.jackson               | jackson-core-asl             | 1.9.13         | []()                                                             |
+| org.codehaus.jackson               | jackson-mapper-asl           | 1.9.13         | []()                                                             |
+| org.mongodb                        | mongo-java-driver            | 3.2.2          | []()                                                             |
+| org.mortbay.jasper                 | apache-el                    | 8.0.33         | []()                                                             |
+| org.springframework.boot           | spring-boot-starter-security | 1.4.7.RELEASE  | []()                                                             |
+| org.springframework.data           | spring-data-mongodb          | 1.9.11.RELEASE | []()                                                             |
+| org.springframework.security.oauth | spring-security-oauth2       | 2.0.14.RELEASE | []()                                                             |
+| org.springframework.security       | spring-security-jwt          | 1.0.5.RELEASE  | []()                                                             |
+| software.amazon.ion                | ion-java                     | 1.0.2          | []()                                                             |
 
+## Test and build dependencies
+
+CQ:
+
+| Group ID            | Artifact ID                 | Version |
+| ------------------- | --------------------------- | ------- |
+| de.flapdoodle.embed | de.flapdoodle.embed.mongo   | 1.50.5  |  |
```

Comment:
Looks strange: Either there is one column to much or the heading is missing

Reply:
Fixed.